tracker-radar-tests/TR-domain-matching/domain_matching_tests.json

```diff
@@ -400,7 +400,10 @@
                 "siteURL": "https://example2a.com",
                 "requestURL": "https://standard1.test/request-1a",
                 "requestType": "stylesheet",
-                "expectAction": "ignore"
+                "expectAction": "ignore",
+                "exceptPlatforms": [
+                    "ios-browser"
```

Comment:
Can we please document those exceptions in https://github.com/duckduckgo/privacy-reference-tests/tree/main/tracker-radar-tests/TR-domain-matching#platform-exceptions-1 ?